client/src/components/Alert/index.tsx

```diff
@@ -0,0 +1,21 @@
+import styles from './style.module.scss';
+import Error from '../../../public/assets/error.svg';
+import { PropsWithChildren } from 'react';
+
+interface AlertProps {
+  marginBottom: 0 | 1 | 1.5 | 2;
+}
+
+const Alert = ({ marginBottom, children }: PropsWithChildren<AlertProps>) => {
+  return (
+    <div className={styles.alert} style={{ marginBottom: `${marginBottom}rem` }}>
+      <div className="error">
+        <Error />
+      </div>
+
+      {children}
+    </div>
+  );
+};
+
+export default Alert;
```

Comment:
If we're doing font stuff here might as well just use Typography. Also, what's the difference between this and useToast from portal-v2? If it's similar, let's just use toasts

Reply:
![image](https://github.com/user-attachments/assets/39d8ef58-fb16-4ce7-a70b-cb0ba76a820a)

From the Figma it doesn't look like a pop up? Let me know if toasts can be non popup or if we want to change the error to a pop up